systemtests/src/main/java/io/enmasse/systemtest/platform/apps/SystemtestsKubernetesApps.java

```diff
@@ -155,18 +160,25 @@
                         "kubernetes"
         };
 
-        POSTGRESQL_EXAMPLE_BASE = Paths.get("../templates/iot/examples/postgresql/deploy");
-        var pgBase = Paths.get("../templates/iot/examples/postgresql/create.sql");
+        POSTGRESQL_EXAMPLE_BASE = examplesIoT.resolve("postgresql/deploy");
+        var pgDevCon = examplesIoT.resolve("postgresql/create.devcon.sql");
+        var pgJsonBase = examplesIoT.resolve("postgresql/create.sql");
```

Comment:
those rust habits :stuck_out_tongue: 
`Path` is easier for maintainability IMHO